templates/taxi/car_detail.html

```diff
@@ -1,7 +1,9 @@
 {% extends "base.html" %}
 
 {% block content %}
-  <p>Manufacturer: ({{ car.manufacturer.name }}, {{ car.manufacturer.country }})</p>
+  <p>Manufacturer: ({{ car.manufacturer.name }}, {{ car.manufacturer.country }})
+    <a style="float: right" href="{% url "taxi:car-update" pk=car.id %}">Update</a><a style="float: right" href="{% url 'taxi:car-delete' pk=car.id%}">Delete</a>
```

Comment:
Code Style: As per the checklist, it's recommended to add a `Cancel` button that leads to the previous page the user was on, apart from the `Update` and `Delete` buttons. This enhances the user experience by providing an option to abort an action.

Reply:
done